dev-qt/qtsvg/qtsvg-6.3.0.ebuild

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qt6-build
+
+DESCRIPTION="SVG rendering library for the Qt6 framework"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+DEPEND="
+	=dev-qt/qtbase-${PV}*
```

Comment:
probably needs `[gui]` at least

Reply:
Could add USE="gui widgets xml", but we didn't have those with Qt5 either... so for now I'll just add them unconditionally.